client/client-multi/client-c/build.sh

```diff
@@ -30,9 +30,13 @@ then
     help
 fi
 
+if [ -z ${MAX_LOG_LEVEL} ]
```

Comment:
The following solution is incorrect: http://stackoverflow.com/a/13864829/2538771